README.md

```diff
@@ -10,6 +10,7 @@ Community (college) maintained list of algorithm implementations and codes.
 |:--------------|:----------------:|:----------------:|:----------------:|:-----------------:|
 | [Linear Search](https://en.wikipedia.org/wiki/Linear_search) | | | | [\[X\]](linear_search/linear_search.py) |
 | [Quicksort](https://en.wikipedia.org/wiki/Quicksort) | [\[X\]](quicksort/quicksort.c) | | | |
+| [Longest Common Subsequence](http://www.geeksforgeeks.org/dynamic-programming-set-4-longest-common-subsequence) | [\[X\]](Longest_Common_Subsequence/main.c) | | | 
```

Comment:
This list should be in alphabetical order. Should LCS will come one place above.